.mailmap

```diff
@@ -513,6 +513,8 @@ Dennis Sweeney <sweeney.427@osu.edu> sweeneyde <sweeney.427@osu.edu>
 Denys Rybalka <rybalka.denis@gmail.com>
 Devang Kulshreshtha <devang.kulshreshtha.cse14@itbhu.ac.in>
 Devansh <be19b002@smail.iitm.ac.in> prototypevito <96943731+prototypevito@users.noreply.github.com>
+Devansh Jaiswal <jaiswaldevansh27@gmail.com>
+Devansh Jaiswal <jaiswaldevansh27@gmail.com> devanshj27 <107506278+devanshj27@users.noreply.github.com>
```

Comment:
you added your mail 2 times. Or ?

Reply:
yes, it was added intentionally to avoid any problems.